files/default/supervisor-ckan-worker.conf

```diff
@@ -13,7 +13,7 @@ command=/usr/lib/ckan/default/bin/paster --plugin=ckan jobs worker --config=/etc
 
 
 ; User the worker runs as.
-user=apache
+user=ckan
```

Comment:
unsure about the user change.

Reply:
The worker is a python process, not Apache. We were running as Apache so that it could easily log to the existing httpd directory, but it turns out that's not a great idea when it needs separate logrotate handling.

Reply:
will it still have access to the old attachments and reading the config file?

Reply:
The config file is publicly readable, and resources are retrieved using the privileges of the account that requested the datastore refresh.